vulnerabilities/importers/nvd.py

```diff
@@ -90,6 +100,28 @@ def extract_summary(cve_item):
         summaries = [desc["value"] for desc in cve_item["cve"]["description"]["description_data"]]
         return max(summaries, key=len)
 
+    @staticmethod
+    def extract_severity_scores(cve_item):
+        severity_scores = []
+
+        if cve_item["impact"].get("baseMetricV3"):
+            severity_scores.append(
```

Comment:
For each CVSS scores we really want to store:
1. a score proper
2. the vector string as another "scoring system" , possibly like `cvssv2_vector` and `cvssv3_vector` that are each their own extra `VulnerabilitySeverity`